README.md

```diff
@@ -353,13 +353,13 @@ See the [Byron specification](https://hydra.iohk.io/job/Cardano/cardano-ledger-s
 
 
 ## Update proposal voting
-You can create and submit byron update proposal votes with the `create-byron-proposal-vote` & `submit-byron-proposal-vote` commands. The following are two example commands:
+You can create and submit byron update proposal votes with the `create-proposal-vote` & `submit-proposal-vote` commands. The following are two example commands:
 
 
 Byron vote creation:
 
 ```
-cabal exec cardano-cli -- byron create-byron-proposal-vote
+cabal exec cardano-cli -- byron create-proposal-vote
```

Comment:
This needs to be changed to `cabal exec cardano-cli -- byron node create-proposal-vote` there is a missing `node` part, the same goes for located above commands for creating and submitting update proposal:
```
cabal exec cardano-cli -- byron node create-update-proposal
cabal exec cardano-cli -- byron node submit-update-proposal
```